Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift

```diff
@@ -16,10 +16,10 @@ struct ChatMessageQuoteAvatarAlignment: Equatable {
 }
 
 /// A view that displays a quoted message.
-public typealias ChatMessageQuoteView = _ChatMessageQuoteView<NoExtraData>
+public typealias MessageQuoteView = _MessageQuoteView<NoExtraData>
```

Comment:
Is this a quote view for a "message" or a `ChatMessage`? 🤔 Maybe the latter?

Reply:
Then should it be `ChatMessageQuoteView` or `QuoteChatMessageView` ? This is why having `Chat` prefix in some views makes It harder to name things, eventually the views won't have consistent and will be confusing IMO 🤔

Reply:
Done ✅